From 621ce5d155618b47f6afd5f6d4450b7236af306b Mon Sep 17 00:00:00 2001
From: Matthew McClintock <msm@freescale.com>
Date: Tue, 7 Sep 2010 08:56:55 +0000
Subject: [build 02/11] arch/powerpc: Remove duplicate/redundant Altivec
 entries
Git-commit: e1fc235d7425c25f4090438afb4df312c5de0d13
Patch-mainline: v3.3
References: bsc#967716

In lieu of having multiple similiar lines, we can just have one
generic cpu-as line for CONFIG_ALTIVEC

Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 arch/powerpc/Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/powerpc/Makefile b/arch/powerpc/Makefile
index c202796..2b6ae1c 100644
--- a/arch/powerpc/Makefile
+++ b/arch/powerpc/Makefile
@@ -131,8 +131,7 @@ KBUILD_CFLAGS		+= -mno-sched-epilog
 endif
 
 cpu-as-$(CONFIG_4xx)		+= -Wa,-m405
-cpu-as-$(CONFIG_6xx)		+= -Wa,-maltivec
-cpu-as-$(CONFIG_POWER4)		+= -Wa,-maltivec
+cpu-as-$(CONFIG_ALTIVEC)	+= -Wa,-maltivec
 cpu-as-$(CONFIG_E500)		+= -Wa,-me500
 cpu-as-$(CONFIG_E200)		+= -Wa,-me200
 
-- 
1.8.5.6

